markdownlint/TOP006_fullFencedCodeLanguage/tests/TOP006_test.md

```diff
@@ -0,0 +1,64 @@
+### Introduction
+
+This file should flag with TOP006 errors, and no other linting errors.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+Valid div due to each tag being surrounded by blank lines.
+
+</div>
+
+#### Custom section
+
+```js
+console.log('This code block should flag an error as it uses "js" instead of "javascript".');
+```
+
+```javascript
+console.log('This code block is valid as it uses the appropriate full name.');
+```
+
+```md
+The rule catches the following languages, as they are they ones expected to be seen in this repo's files
+md => markdown
+rb => ruby
+js => javascript
+txt => text
+```
+
+```rb
+puts "Example of rb flagging."
+```
+
+```ruby
+puts "Use the full name!"
+```
+
+```txt
+As does txt.
+```
+
+```html
+HTML is not considered as only the abbreviated name is a valid option.
+The same applies to similar languages like CSS.
+```
+
```

Comment:
Can we just add in a css code block as well?

Reply:
Threw in a jsx one too for good measure!